### DISCUSSION TITLE
Crazyflie bolt1.1 hardware

### DISCUSSION THREAD

**Discussion by CasperLiZ** [bitcraze/discussions#978] [Q&A]
I'm sorry to bother you. I made a flight control board according to the schematic diagram of bolt1.1, but I found that an EEPROM sensor was connected to the NRF chip, and after burning the code, it was empty, indicating that there was a problem. However, after removing the EEPROM chip, some results appeared. Please tell me how I can solve it. Thank you very much.

SYS: Build 32:9343aa686600 (2023.07 +32) MODIFIED
SYS: I am 0x38313032353247070031001E and I have 1024KB of flash!
CFGBLK: v1, verification [OK]
DECK_CORE: 0 deck(s) found
IMU: BMI088: Using SPI interface.
IMU: BMI088 Gyro connection [OK].
IMU: BMI088 Accel connection [OK]
IMU: BMP388 I2C connection [OK]
ESTIMATOR: Using Complementary (1) estimator
CONTROLLER: Using PID (1) controller
MTR-DRV: Using brushless motor driver
SYS: About to run tests in system.c.
SYS: NRF51 version: 2023.02*
EEPROM: I2C connection [OK].
STORAGE: Storage check [OK].
IMU: BMI088 gyro self-test [OK]
SYS: Self test passed!
STAB: Wait for sensor calibration...
SYS: Free heap: 20496 bytes
STAB: Starting stabilizer loop
SUP: Can not fly
STAB: WARNING: stabilizer loop rate is off (996)

**Comment by tobbeanton** [comment]
The EEPROM is connected to the STM32 and not the NRF. I don't really see what is wrong?

**Reply by CasperLiZ** [reply]
Sorry, is the problem I described, in the NRF section 14 pin P0.08 connected to a DS28E05 chip, as far as I know it is a memory. When I first started it said "Memory error: wrong header ID", so I removed it, but the above problem came up, so can you tell me, what does this do? How do I solve "Memory error" and the problems described above. Thank you very much. Have a good day.
![image](https://github.com/bitcraze/discussions/assets/51102143/cbe437fe-4c79-499d-bb2f-7a2e404effbd)

**Reply by CasperLiZ** [reply]
I removed the DS28E05 chip with a heat gun.

**Reply by tobbeanton** [reply]
The DS28E05 is not used on the Bolt so that is why, it should not be there. So what do you plan to do with you own board?

**Reply by CasperLiZ** [reply]
Thanks for your reply. I plan to use this board to install a computer for path planning. Compared with CF21, this scheme can provide me with better motivation.
But I am still unable to take off. The log is as follows:
SYS: Build 32:9343aa686600 (2023.07 +32) MODIFIED
SYS: I am 0x38313032353247070031001E and I have 1024KB of flash!
CFGBLK: v1, verification [OK]
DECK_CORE: 0 deck(s) found
IMU: BMI088: Using SPI interface.
IMU: BMI088 Gyro connection [OK].
IMU: BMI088 Accel connection [OK]
IMU: BMP388 I2C connection [OK]
ESTIMATOR: Using Complementary (1) estimator
CONTROLLER: Using PID (1) controller
MTR-DRV: Using brushless motor driver
SYS: About to run tests in system.c.
SYS: NRF51 version: 2023.02*
EEPROM: I2C connection [OK].
STORAGE: Storage check [OK].
IMU: BMI088 gyro self-test [OK]
SYS: Self test passed!
STAB: Wait for sensor calibration...
SYS: Free heap: 20496 bytes
STAB: Starting stabilizer loop
SUP: Can not fly
STAB: WARNING: stabilizer loop rate is off (996)
There is no "Ready to fly", because all the self-testing is OK. Do you want to know what else causes this problem and what do I need to operate on cfclient?

**Reply by tobbeanton** [reply]
Are your running the bolt platform?

> Compared with CF21, this scheme can provide me with better motivation.

Sorry, don't really understand this sentence?

**Reply by CasperLiZ** [reply]
Sorry for my expression problem, my English is not very good, so I will use translation software to communicate with you. I use bolt because it can support 250, 450, 550, etc. rack sizes, and I need a lot more peripherals than using a BigQuad deck.

**Reply by amitx64** [reply]
how rewrite bolt `ow memory`.  I just want to know `board name`, ` vid` and `pid`
